### PR TITLE
Replaced headings with abbreviations - OSSP-legend.js

### PR DESCRIPTION
**Pull request by jobyko**
@panickervinod  @asa1997  @sumodgeorge 

Please have a look and share your comments.

### CODE REVIEW

File: bes_theme/assets/data/OSSP-legend.js [hunk 1]
```diff
@@ -0,0 +1,567 @@
...
+                "url": "https://api.github.com/repos/Be-Secure/BeSLighthouse/labels/TD-C-Wlt",
+                "name": "TD-C-Wlt",
+                "color": "749812",
+                "default": false,
+                "description": "Technology  Domain  Classifier: Composition:  Type :  Wallet App"
+            }
+        },
+
+        "COM": {
+            "COM - C" : {
```
Comment (panickervinod) [inline bes_theme/assets/data/OSSP-legend.js:535]: @asa1997 Each foundation must be a separate label , the structure should change to reflect the same

File: bes_theme/assets/data/OSSP-legend.js [hunk 2]
```diff
@@ -0,0 +1,567 @@
+[
+  {
+    "TOSS-PM": {
+
+        "bes_oss_tech_stack": {
+          "DO" : {
+                  "id": 3333824398,
+                  "node_id": "MDU6TGFiZWwzMzMzODI0Mzk4",
```
Comment (panickervinod) [inline bes_theme/assets/data/OSSP-legend.js:8]: @jobyko @asa1997  only "id" & possibility "name" or "url"  is required as other details are present in labels json ,

**Comment by panickervinod** [comment]
@jobyko @asa1997 Please address the following.

Each foundation must be a separate label , the structure should change to reflect the same 

only "id" & possibility "name" or "url" is required as other details are present in labels json

File: bes_theme/assets/data/OSSP-legend.js (hunk 1 above)
Comment (jobyko) [reply bes_theme/assets/data/OSSP-legend.js:535]: At the moment, foundation labels are not available.
Will be raising PR  updated with newly created foundation based labels.

File: bes_theme/assets/data/OSSP-legend.js (hunk 2 above)
Comment (jobyko) [reply bes_theme/assets/data/OSSP-legend.js:8]: Will keep, id and name alone for the headings in the upcoming PR

**Pull request closed without merging**